content/influxdb/cloud-dedicated/_index.md

```diff
@@ -21,14 +21,13 @@ time series use case.
 <a class="btn" href="{{< dedicated-link >}}">Request an InfluxDB Cloud Dedicated cluster</a>  
 <a class="btn" href="/influxdb/cloud-dedicated/get-started/">Get started with InfluxDB Cloud Dedicated</a>
 
-## The InfluxDB IOx storage engine
+## InfluxDB 3.0
```

Comment:
"3.0" and not "v3"? Do we need to change that elsewhere?

Reply:
There's still some debate on it. I figure we can come back and change it whenever things are finalized and the dust settles.